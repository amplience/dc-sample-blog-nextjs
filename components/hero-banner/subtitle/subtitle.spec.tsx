import React from 'react';
import renderer from 'react-test-renderer';
import HeroBannerSubtitle from './subtitle';

describe('HeroBannerSubtitle', () => {
  test('renders hero banner subtitle', async () => {
    const props = {
      subTitle: 'Test subtitle'
    };
    const component = await renderer.create(<HeroBannerSubtitle {...props} />);
    expect(component.toJSON()).toMatchSnapshot();
  });

  test('renders nothing when there is no subtitle', async () => {
    const props = {};
    const component = await renderer.create(<HeroBannerSubtitle {...props} />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
