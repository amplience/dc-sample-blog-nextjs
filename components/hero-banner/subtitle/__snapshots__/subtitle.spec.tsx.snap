// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeroBannerSubtitle renders hero banner subtitle 1`] = `
<h2
  className="jsx-1054465095"
>
  Test subtitle
</h2>
`;

exports[`HeroBannerSubtitle renders nothing when there is no subtitle 1`] = `""`;
